src/features/wallet/root/BalanceCard.tsx

```diff
@@ -47,7 +57,7 @@ const BalanceCard = ({ onReceivePress, onSendPress }: Props) => {
               opacity={0.4}
               lineHeight={25}
             >
-              .45876891 HNT
+              .{decimalPart}
             </Text>
```

Comment:
Should we update the account DC and HST balance?

Reply:
I was going to do that in a separate PR because it'll affect the animation points of the bottom sheet, but yes we should

Reply:
Ok no problem, sounds good